controllers/suite/clusters/humiocluster_controller_test.go

```diff
@@ -19,11 +19,12 @@ package clusters
 import (
 	"context"
 	"fmt"
-	cmapi "github.com/cert-manager/cert-manager/pkg/apis/certmanager/v1"
 	"os"
 	"reflect"
 	"strings"
 
+	cmapi "github.com/cert-manager/cert-manager/pkg/apis/certmanager/v1"
+
```

Comment:
Not sure why this is here? Seems completely unrelated to this work.

Reply:
I do not have any clue why this ended up here as I was not working on this file